src/pages/documentation/release-notes/index.md

```diff
@@ -22,6 +22,23 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 * `target.previewEnabled` configuration setting is no longer required to enable preview mode in Target iOS SDK. If not configured, the default value here will be set to true.
 
+### AEP Flutter Plugins 2.0.0
+
+* Updated all the AEP Flutter plugins to use MobileCore for Android 2.x and compatible extensions.
```

Comment:
nit: reword "to use MobileCore for Android 2.x" -> to use MobileCore 2.x for Android